helpers/FileManager.go

```diff
@@ -0,0 +1,16 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"strings"
+)
+
+func ReadFile(filename string) (rows []string, err error) {
+	content, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return
```

Comment:
When working with errors, the more explicit you could be, the better.

It's a good practice to start wrapping errors, to better trace them when something goes wrong.

Give it a look to errors.New or fmt.Errorf methods.